smarty/templates/instrument_html.tpl

```diff
@@ -237,7 +237,7 @@
 				{else}
 					{if $inTable eq "FALSE"}
 						{assign var="inTable" value="TRUE"}
-						<table class="table table-instrument">
+						<table class="table table-instrument" style="white-space: nowrap;">
```

Comment:
there seems to already be a table-instrument css class, why not add the property there?